node/store_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package node
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
question: These tests do not appear to actually test their config works, only that it does not error - they (the WithFoo funcs) could be empty funcs for all these tests care.  Do you think they are worth keeping and maintaining?

Reply:
I've updated the tests. They should make more sense now.